### ISSUE TITLE
Evaluate and improve map accessibility/colours

### ISSUE THREAD

**Issue by tobypinder** [octoreso/octoreso#7]
Gather feedback on the new icon set, see if it's useful enough.

**Comment by Jurugu00** [comment]
Colours of missions that are near together could have more distinct colours, so that they are more recognizable. It would be great to have some mouse-over features, so that i hover over a portal the missions will be highlighted in the mission list. On the over hand, if i hover over a certain mission in the list, all its portals could be highlighted in the map, so that it is easy to get an overview.

**Comment by tobypinder** [comment]
I agree on the mouseover comments, I will extract this to a separate issue. 

I think I can improve the colours - right now the colouring algorithm is using a mission's ID directly to derive the colour (and they are sorted by hue, so colour 3 is very close to colour 2). If I hash these values before using them I _should_ get a more random (but consistent) set of colours for the missions/mission series etc.
